js/hw2.js

```diff
@@ -0,0 +1,195 @@
+function getSum() {
+    let from = Number(document.getElementById('task1_from').value);
+    let to = Number(document.getElementById('task1_to').value);
+
+    let result = 0;
+
+    if (from <= to) {
+        for (let i = from; i <= to; i++) {
+            result += i;
+        }
+    }
+    else {
+        for (let i = from; i >= to; i--) {
+            result += i;
+        }
+    }
+
+    document.getElementById("task1_result").innerHTML = result;
+}
+
+function getDivider() {
+    let num1 = Number(document.getElementById('task2_num1').value);
+    let num2 = Number(document.getElementById('task2_num2').value);
+
+    let result = 0;
+
+    (num2 > num1) ? result = num1 : result = num2;
+
+    // we can add check that result > 0 but both numbers should be divided to 1
+    // but it will not work if numbers will be float, hope for the best this time
```

Comment:
You can `Math.floor()` floated numbers and work with integers

Reply:
Could be a workaround, but I don't believe that this approach is correct from a math perspective. As this is a learning task I suppose the assumption that input will be a not float number is ok.